ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-ec-x86.yaml

```diff
@@ -21,7 +21,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: perfscale-rosa-multiaz-ec-data-path-9nodes-periodic
```

Comment:
If we are taking out ec from the test names can we also remove from file name?

Reply:
@paigerube14 I removed it from test name because we have it in file name. Otherwise we will not be able to distinguish between nightly and ec in our "Prow Runs" sheet.